client/src/modules/reports/reports.routes.js

```diff
@@ -9,7 +9,7 @@ angular.module('bhima.routes')
       'aged_creditors',
       'aged_debtors',
       'aggregated_stock_consumption',
-      'analysis_auxiliary_cashboxes',
+      'analysis_auxiliary_cashbox',
```

Comment:
This breaks the report key at IMCK.  You'll need to write a migration script that updates the `report` table  and the key in the `unit` table:

![image](https://user-images.githubusercontent.com/896472/149290185-45126140-3602-40e3-a1de-95ba6ed90194.png)

Reply:
Thanks for noticing